testserver/manage.py

```diff
@@ -3,6 +3,15 @@
 import sys
 
 if __name__ == "__main__":
+    
+    if 'python3' not in sys.executable:  # might be better to check version also.
+    # if sys.hexversion < 0x03000000:
+        import distutils.spawn
+        python3_path = distutils.spawn.find_executable('python3')
+        if python3_path is not None:
+            os.execvp(sys.executable + '3', [sys.executable + '3'] + sys.argv)
+            #TODO what to do if it is None?
```

Comment:
`print('Batavia requires Python 3', file=sys.stderr)`
Is not compatible with Python 2. So I changed it too:
`sys.stderr.write('Batavia requires Python 3' + os.linesep)`